cookiecutter/compat.py

```diff
@@ -131,4 +131,18 @@ def _access_check(fn, mode):
                         return name
         return None
 
+try:
+    isinstance("", basestring)
+
+    def isstr(s):
+        """Determine if an object is a string in Python 2."""
+        return isinstance(s, basestring)
+
+except NameError:
+
+    def isstr(s):
+        """Determine if an object is a string in Python 3."""
+        return isinstance(s, str)
```

Comment:
@jacebrowning Since this module already knows the version of python, what do you think of:
```python
def is_str(s):
    return isinstance(s, str if PY3 else basestring)
```
?